xml/ha_qdevice-qnetd.xml

```diff
@@ -0,0 +1,508 @@
+                  and there it's used in calculations to determine which partition
+                  should be quorate.
+               </para>
+            </listitem>
+         </varlistentry>
+         <varlistentry>
+            <term>Tiebreaker</term>
+            <listitem>
+               <para>
+                  It is used as a fallback if the cluster partitions are completely
+                  equal even with the same heuristics results. It can be configured
+                  to be lowest, higest or specific nodeid.
+               </para>
+            </listitem>
+         </varlistentry>
+      </variablelist>
+   </sect1>
+
+   <sect1 xml:id="sec-ha-qdevice-require">
+      <title>Requirements and Prerequisites</title>
+      <para>
+         Before you start using &qdevice; and &qnet;, make sure the following
+         conditions are fullfilled:
+      </para>
+      <itemizedlist>
+         <listitem>
+            <remark>toms 2020-05-14: See also bsc#1171681</remark>
+            <para>
+               An additional machine which will become the &qnet; server with
+               &sls; and <package>corosync-qnetd</package> package installed.
+            </para>
+         </listitem>
+         <listitem>
+            <para>
+               A network address used by the &qnet; server which can be neutrally
+               reached by all cluster nodes.
+            </para>
+         </listitem>
+      </itemizedlist>
+   </sect1>
+
+   <sect1 xml:id="sec-ha-qdevice-setup-qnetd">
+      <title>Setting up the &qnet; Server</title>
+      <para>
+         The &qnet; server is not part of the cluster stack, it is also
+         not a real member of your cluster. As such, you cannot move resources
+         to this server.
+      </para>
+      <para>
+         As the &qnet; server is <quote>state free</quote>, you do not usually
+         need to configure anything (although you could in <filename>/etc/sysconfig/corosync-qnetd</filename>).
+         The <package>corosync-qnetd</package> service by default runs the daemon
+         as user <systemitem class="username">coroqnetd</systemitem>
+         in group <systemitem class="groupname">coroqnetd</systemitem>. This avoids
+         to run the daemon as &rootuser;.
+      </para>
+      <para>
+         To create a &qnet; server, proceed as follows:
+      </para>
+      <procedure>
+         <step>
+            <para>
+               On the machine that will become the  &qnet; server, install
+               &sls; &productnumber;.
+               <remark>toms 2020-05-15: See also bsc#1171681</remark>
+            </para>
+         </step>
+         <step>
+            <para>
+               Log in to the &qnet; server and install the following package:
+            </para>
+            <screen>&prompt.root;<command>zypper</command> install corosync-qnetd</screen>
+         </step>
+         <step>
+            <para>
+               Start and enable the service:
```

Comment:
Actually we could not manually start and enable the qnetd.service specifically
On the cluster node, bootstrap process will do that during setup qdevice process

Reply:
Exactly, in our openQA tests, we basically only install the package and we let the bootstrap script start and enable the qnetd service in the qnetd server.

Reply:
> Actually we could not manually start and enable the qnetd.service specifically

It reads "we don't need to manually start and enable qnetd.service specifically" :-)

Reply:
Ok, thanks for the hint. In that case, I will remove this step. However, that leaves the procedure with only two steps which is a bit meager. I've added an additional sentence:

```xml
You do not need to manually start the <systemitem class="daemon"
>corosync-qnetd</systemitem> service. The bootstrap scripts
will take care of the startup process during the qdevice stage.
```